src/frontend/src/primitives/H.tsx

```diff
@@ -1,24 +1,14 @@
-import type { HTMLAttributes } from 'react'
-import classNames from 'classnames'
-import { css } from '@/styled-system/css'
+import { Text } from './Text'
 
-export const H1 = ({
+export const H = ({
   children,
-  className,
+  lvl,
   ...props
-}: HTMLAttributes<HTMLHeadingElement>) => {
+}: React.HTMLAttributes<HTMLHeadingElement> & { lvl: 1 | 2 | 3 }) => {
```

Comment:
out of curiosity, why having only 1,2,3?

Reply:
I didn't setup any actual styles for level 4 headings and more, waiting to see if we have any